packages/api-explorer/__tests__/JsonForm/index.test.jsx

```diff
@@ -33,15 +42,18 @@ describe('JSONForm ', () => {
    
     const originalDateNow = Date.now
     const originalRandom = Math.random
+    const originalRequestAnimationFrame = window.requestAnimationFrame
 
     Date.now = jest.fn().mockReturnValue(1589886640576)
     Math.random = jest.fn().mockReturnValue(0.5)
+    window.requestAnimationFrame = jest.fn()
```

Comment:
Can we check at least that the function is called? and with what arguments?

Reply:
it's called with a function as callback.
This mock is done only in order to prevent the snapshot to fail because of what json-editor does inside this function (that with snapshot seems to be called randomly).

We could add a comment to explain this, but I'd not make an expect because it is not the business of this test assert that the function is called. Adding that expect would make this test less clear.

Reply:
(and if the function for some reason would not be called, the test should not fail

Reply:
@fredmaggiowski what do you think about what @dibericky wrote? 
The point is that before adding that line of code the test had a not deterministic behaviour (1 time out of 10 it ends with a fail). That line is useful to avoid this unwanted behaviour and to make the test work as intended, therefore I don't think it is relevant to verify if this function is called or not, let me know!

Reply:
Let's just document it then!

Reply:
Done!